### DISCUSSION TITLE
23.12.20 데일리 스크럼

### DISCUSSION THREAD

**Discussion by Jeonwoojae** [petit-a-petit/MIML#94] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 리드미 머지
    - 위키 작성 (패키지 구조)
- 내일은 어떤 일을 하나요?
    - nGrinder 설치 및 사용법 학습

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 프로젝트 문서화
    - 음악 조회 api 내용 추가
  - 코드 리뷰
  - batch 작업 테스트 코드 작성
- 내일은 어떤 일을 하나요?
  - PR merge
  - 리팩토링 고민
